golang/go-cloud-run-hello-world/index.html

```diff
@@ -164,12 +39,30 @@ <h2>Congratulations, you successfully deployed a container image to Cloud Run</h
 			</p>
 
 			<h3>What's next?</h3>
-			<p class="next">Manage Cloud APIs and libraries in Cloud Code for <a href="https://cloud.google.com/code/docs/vscode/client-libraries" target="_blank" rel="noopener">VSCode</a> or <a href="https://cloud.google.com/code/docs/intellij/client-libraries" target="_blank" rel="noopener">IntelliJ</a></p>
-			<p class="next">Explore more options with
+			<ul>
+				<li class="next">
```

Comment:
I recommend that we regroup the content in these two sections to differentiate betweeen things to try in the product, and how to engage with us. I'm not sure if the message is 100% correct, so please verify.

Things to try:
- Debugging your Cloud Run service with Cloud Code
- Make your application more secure using Secret Manager
- Learn how Cloud Code can accelerate your application development
- Learn how Cloud Run makes your development experience simpler


Want to help shape the future of Cloud Code?
- Sign up to participate in our ongoing research studies
- Join the #cloudcode channel in the Google Cloud Platform Slack community to ask questions, provide feedback, and get the latest Cloud Code news.

Reply:
Good idea! I've changed the wording a bit, let me know what you think:
<img width="734" alt="cloud-run-content-update" src="https://user-images.githubusercontent.com/38271546/106339321-13d92800-624b-11eb-9363-9c2f4e864a7a.png">